src/components/HorizontalStepper/HorizontalStepper.module.css

```diff
@@ -10,7 +10,6 @@
 .horizontal-stepper {
   display: flex;
   align-items: center;
-  justify-content: space-between;
```

Comment:
Was this not needed?

Reply:
It was apparently not. Behaving the same in the stories and page stories